content/docs/reference-test-renderer.md

```diff
@@ -19,6 +19,8 @@ This package provides a React renderer that can be used to render React componen
 
 Essentially, this package makes it easy to grab a snapshot of the platform view hierarchy (similar to a DOM tree) rendered by a React DOM or React Native component without using a browser or [jsdom](https://github.com/tmpvar/jsdom).
 
+**Note:** make sure `react-test-renderer` version is the same with `react`, `react-dom` version.
```

Comment:
What about something like:

```md
> Note
>
> To prevent potential incompatibilities, ensure that `react-test-renderer` and other `react` packages all use the same version (eg `react@16.2.0`, `react-dom@16.2.0`, `react-test-renderer@16.2.0`).
```

As Dan mentioned in the main PR, this format will style differently and be more noticeable:
![screen shot 2017-12-20 at 9 10 14 am](https://user-images.githubusercontent.com/29597/34219302-9cbbafbe-e565-11e7-981f-cf646ba88bb2.png)

Reply:
We can also add this note in other places too, since it's pretty generic. 😄

Reply:
@bvaughn  It is so clean and easy to understand. 😄

Reply:
Ok 😄 Want to update this PR to use that wording then?

Reply:
Sure I have just updated the new styles & words. 😁

Reply:
any suggestion to put this to another place in document?

Reply:
I also suggest to put this to: https://reactjs.org/tutorial/tutorial.html before "Help, I’m Stuck!" section

Reply:
Looks great!

Maybe we could also add it somewhere on pages like:
* /docs/react-dom.html
* /docs/shallow-renderer.html
* /docs/test-utils.html

Not sure if it's necessary to list specified things on the "Help, I'm Stuck!" section since there are so many potential causes. Seems better to refer to other resources (like it does).